packages/forma-36-website/src/pages/index.jsx

```diff
@@ -64,6 +64,25 @@ function IndexPage() {
           and learn how to do it
         </Paragraph>
       </Section>
+      <Section>
+        <DisplayText>Give us feedback!</DisplayText>
+        <Paragraph>
+          Use one of the the templates on{' '}
+          <TextLink
+            href="https://github.com/contentful/forma-36/issues/new/choose"
+            target="_blank"
+          >
+            GitHub
+          </TextLink>
+        </Paragraph>
+        <Paragraph>
+          or use this anonymous{' '}
+          <TextLink href="https://forms.gle/qC7LLbiy4CcF5HPLA" target="_blank">
```

Comment:
shouldn't this also have the `rel="noopener noreferrer`?

Reply:
good point! fixing